accepted/2022/version-selection.md

```diff
@@ -0,0 +1,449 @@
+# .NET 7 Version Selection Improvements
+
+**Owner** [Rich Lander](https://github.com/richlander)
+
+Each development platform is based on some composition model for platform versions (where they are installed, private vs global installation/visibility, how they are selected, ...). On one hand, there are platforms like Node.js that have a single version in scope at a time (tools like [`nvm`](https://github.com/nvm-sh/nvm) make that more manageable) and on the other we have .NET that allows for multiple versions in scope at a time (managed by the [`dotnet`](https://docs.microsoft.com/dotnet/core/tools/dotnet) tool and MSBuild). .NET offers a multiple ways of selecting a runtime or SDK version, however, there are some inconsistencies across the various gestures and missing capabilities. This document proposes improvements to .NET version selection.
+
+Summary of significant changes:
+
+- Disable multi-level lookup.
+- Change default for `global.json` SDK version to a floor version, as opposed to the current pinning behavior.
+- Enable controlling SDK version by environment variable (ENV).
+- Enable proxying to a local/private SDK via the `PATH` SDK.
+- Enable specifying a floor runtime version for an environment.
+
+Note: The two of those changes are more speculative (less likely to happen).
```

Comment:
Not sure about "The two of those". First two? Last two?

Reply:
Ooops!

```suggestion
Note: The last two of those changes are more speculative (less likely to happen).
```